app/views/operations/new.html.erb

```diff
@@ -1,13 +1,11 @@
-<%= form_with model: @operation, url: "/categories/#{@category.id}/operations" do |form| %>
+<%= form_with model: @operation, url: "/categories/#{@category.id}/operations", class: 'new-form' do |form| %>
   <div class='field'>
-    <%= form.label :name %>
-    <%= form.text_field :name %>
+    <%= form.text_field :name, placeholder: 'Name' %>
   </div>
   <div class='field'>
-    <%= form.label :amount %>
-    <%= form.number_field :amount, step: 0.1 %>
+    <%= form.number_field :amount, step: 0.1, placeholder: 'Amount' %>
   </div>
   <div class='field'>
-    <%= form.submit 'Add' %>
+    <%= form.submit 'Add', class: 'add-button' %>
   </div>
 <% end %>
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that For "Add a new transaction" page Kindly make sure that The user fills out a form to create a new transaction with:
* name (mandatory)
* amount (mandatory)
*** categories (mandatory at least one)** 

- But Your App doesn't have **categories (mandatory at least one)** Kindly Update it.
![image](https://github.com/Movgto/ror-budget-app/assets/93598189/380a8cf1-786a-480b-a680-9dad28c8a6f0)